### DISCUSSION TITLE
Timeframe for CVE-2023-22102 resolution?

### DISCUSSION THREAD

**Discussion by atziatzios-ccycloud** [jruby/activerecord-jdbc-adapter#1161] [Q&A] [answered]
Hi. I just wanted to highlight an already raised issue (https://nvd.nist.gov/vuln/detail/CVE-2023-22102) with the mysql adapter version in use in the latest release. 

How complex/problematic is an update to a non vulnerable version of the adapter?

Thanks to anyone taking the time to answer.

**Comment by headius** [comment]
Aha, thanks for bringing it to our attention. I'm going to switch this to an issue and we'll get a release of jdbc-mysql out with updated driver.

**Comment by headius** [comment, marked as answer]
https://github.com/jruby/activerecord-jdbc-adapter/issues/1162
